src/games/strategy/triplea/printgenerator/UnitInformation.java

```diff
@@ -82,10 +82,8 @@ protected void saveToFile(final PrintGenerationData printData, final Map<UnitTyp
       unitInformation.write("\r\n");
       unitInformation.close();
     } catch (final FileNotFoundException e) {
-      // TODO Auto-generated catch block
```

Comment:
Bad error handling again here, we shoudl be printing out the absolute path of the file that we failed to find.